mongo/datadog_checks/mongo/config.py

```diff
@@ -99,6 +99,14 @@ def __init__(self, instance, log):
         self.service_check_tags = self._compute_service_check_tags()
         self.metric_tags = self._compute_metric_tags()
 
+        # DBM config options
+        self.dbm_enabled = is_affirmative(instance.get('dbm', False))
```

Comment:
What does this on/off switch control right now?

Reply:
The `dbm` flag is used to control database hosts showing up on `Database Monitoring` host panel. It will also be used to control additional dbm related features in the integration such as sampling or explain plans.